internal/lefthook/runner/runner.go

```diff
@@ -222,7 +222,8 @@ func (r *Runner) runScript(script *config.Script, unquotedPath string, file os.F
 		args = strings.Split(script.Runner, " ")
 	}
 
-	quotedScriptPath := shellescape.Quote(unquotedPath)
+	quotedScriptPath := fmt.Sprintf("%#v", unquotedPath)
```

Comment:
Wow, so if I correctly understood, with `Sprintf("%#v")` the bug doesn't reproduce, right? Interesting :thinking: 
You know, a few releases before the path for scripts wasn't even quoted. I think this is a regression, and I added the quoting accidentally. 

@stonesbg could you remove [this](https://github.com/stonesbg/lefthook/blob/master/internal/lefthook/runner/runner.go#L225) line and rename the `unquotedPath` argument to be just `path`?

Good job with finding the reason for the buggy behavior! I appreciate it so much!

Reply:
Not at all :) Let me help